rules/docker-iccpd.mk

```diff
@@ -0,0 +1,16 @@
+# docker image for iccpd agent
+
+DOCKER_ICCPD = docker-iccpd.gz
```

Comment:
Please have configurable option to enable/disable docker build. Please check other docker example like NAT/SFLOW

Reply:
Thanks,done